creusot-contracts/src/logic/ord.rs

```diff
@@ -202,41 +211,7 @@ impl OrdLogic for bool {
         }
     }
 
-    #[ghost]
```

Comment:
Reminds me we need to change the behavior of default methods so that we don't need to repeat them, they aren't very default today..